### PR TITLE
Warn for head position nan for rc 63

### PR DESCRIPTION
**Pull request by sethalves**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6630/app_12270_PR.apk

**Comment by sethalves** [comment]
to test:
- should be no functional change.
- pay attention to the avatar and other avatars, watch for anomalies
- try 3rd person camera and mirror camera

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6631/app_12270_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6631/HighFidelity-Beta-PR12270-2962c8979ccde39c2666251b463dd993c4e910d7.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6631/HighFidelity-Beta-PR12270-2962c8979ccde39c2666251b463dd993c4e910d7.dmg

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -2700,27 +2706,42 @@ void MyAvatar::setWalkSpeed(float value) {
 }
 
 glm::vec3 MyAvatar::getPositionForAudio() {
+    glm::vec3 result;
     switch (_audioListenerMode) {
         case AudioListenerMode::FROM_HEAD:
-            return getHead()->getPosition();
+            result = getHead()->getPosition();
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.cpp]: break;

File: interface/src/avatar/MyAvatar.cpp [hunk 2]
```diff
@@ -2700,27 +2706,42 @@ void MyAvatar::setWalkSpeed(float value) {
...
     switch (_audioListenerMode) {
         case AudioListenerMode::FROM_HEAD:
-            return getHead()->getFinalOrientationInWorldFrame();
+            result = getHead()->getFinalOrientationInWorldFrame();
         case AudioListenerMode::FROM_CAMERA:
-            return qApp->getCamera().getOrientation();
+            result = qApp->getCamera().getOrientation();
         case AudioListenerMode::CUSTOM:
-            return _customListenOrientation;
+            result = _customListenOrientation;
```
Comment (hyperlogic) [inline interface/src/avatar/MyAvatar.cpp]: break;

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/6634/app_12270_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/6634/HighFidelity-Beta-PR12270-503175cd28146ce017f4b70609370d0863a0dfdc.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/6634/HighFidelity-Beta-PR12270-503175cd28146ce017f4b70609370d0863a0dfdc.dmg

**Comment by Qliemillar** [comment]
Currently being tested

**Comment by Qliemillar** [comment]
Testing completed - PASS @conklin94122

**Pull request merged**